typescript/infra/src/config/helloworld.ts

```diff
@@ -7,12 +7,14 @@ export interface HelloWorldKathyConfig<Chain extends ChainName> {
   runEnv: string;
   namespace: string;
   chainsToSkip: Chain[];
+  /** Whether to cycle once through all pairwise chains, or run kathy as a forever running service */
```

Comment:
I would almost prefer something like:

```ts
runningConfig: { mode: 'cycleOnce' } | {  mode: 'service', fullCycleTime: number, messageSendTimeout: number ... }
```

The current approach makes it really easy to forget something. Will leave final choice up to you since I know we have defaults being set.